controllers/operatorpolicy_status.go

```diff
@@ -669,12 +675,16 @@ var installPlansNoApprovals = metav1.Condition{
 
 // installPlanUpgradeCond is a NonCompliant condition with Reason 'InstallPlanRequiresApproval'
 // and a message detailing which possible updates are available
-func installPlanUpgradeCond(versions []string, approvableIPs []unstructured.Unstructured) metav1.Condition {
+func installPlanUpgradeCond(
+	complianceConfig policyv1beta1.ComplianceConfigAction,
+	versions []string,
+	approvableIPs []unstructured.Unstructured,
+) metav1.Condition {
 	// FUTURE: check policy.spec.statusConfig.upgradesAvailable to determine `compliant`.
 	// For now this condition assumes it is set to 'NonCompliant'
+	canModifyComp := complianceConfig == "NonCompliant"
```

Comment:
I think I'd prefer using `complianceConfig == "NonCompliant"` directly in the condition below, it might make it easier in the future if we add other options. Also just a reminder to remove the `// FUTURE` comment.